gui/slick/views/home.mako

```diff
@@ -392,6 +387,9 @@ $(document).ready(function(){
           % endif
         });
 
+        // Hides size column for now until we can fix it
+        $('[data-show-size]').hide();
+        $('[data-column="6"]').hide();
```

Comment:
Well it hides the columns after they've loaded, just not sure if we should hot-fix it like this or actually remove it for now?